tests/unittests/PartitionerTest.cpp

```diff
@@ -410,10 +449,11 @@ TEST_F(PartitionerTest, Basic1Roofline) {
       {"fc_dot4", 5120},
       {"fc_add_bias4", 96},
   };
-  for (auto const &p : nodeNamesMap) {
-    auto *N = p.first;
-    EXPECT_EQ(myPartitioner.getComputeTime(N), expectedComputeTime[p.second]);
-  }
+  // for (auto const &p : nodeNamesMap) {
+  //   auto *N = p.first;
+  //   EXPECT_EQ(myPartitioner.getComputeTime(N),
+  //   expectedComputeTime[p.second]);
+  // }
```

Comment:
I think this is part is testing the roofline. Why do you comment it out?

Reply:
fixed now